### PR TITLE
Save keyPair to local storage

### PR DESCRIPTION
**Pull request by LaurenSpiegel**
Addresses https://github.com/OpenMined/covid-alert/issues/8

### CODE REVIEW

**Comment by cereallarceny** [comment]
@LaurenSpiegel  I'm going to have @SETIADEEPANSHU now review all PR's for this project. It's in his hands. :)

**Comment by s0l0ist** [comment]
Unfortunately, this implementation is not complete. We need to instantiate the instance from their class constructors after deserialization.

I've fixed this in my in #17 , however, I do not implement encrypted storage at the moment. Will keep this open until merging.

**Comment by LaurenSpiegel** [comment]
@s0l0ist , you cannot have an async function in a constructor. What is the issue with including in componentDidMount? The keys are available when needed.

File: covidalert/src/App.js [hunk 1]
```diff
@@ -55,6 +52,18 @@ export default class extends Component {
     await this.verify();
   };
 
+  setUpKeyPair = async () => {
+    let keyPair = await retrieveItem('keyPair');
+    if (keyPair) {
+      keyPair = JSON.parse(keyPair);
```
Comment (s0l0ist) [inline covidalert/src/App.js:58]: This line needs to also parse the keys directly and create the instance. The current method doesn't attach the instance methods needed for the keys later on in the code.

Comment (LaurenSpiegel) [reply covidalert/src/App.js]: Thank you. I fixed this.

**Comment by cereallarceny** [comment]
Closing in favor of https://github.com/OpenMined/covid-alert/pull/17

**Pull request closed without merging**